people/zuzana.json

```diff
@@ -0,0 +1,21 @@
+{
+
+  "key": "zuzana",
+
+  "name": "BcA. Zuzana Červínová",
+
+  "nick": "Zuzana",
+
+  "email": "zuzana.cervinova@mild.blue",
+
+  "title_cs": "JUNIOR FRONTEND DEVELOPER",
+
+  "title_en": "JUNIOR FRONTEND DEVELOPER",
+
+  "short_desc_en": "Zuzana is a Bachelor of Arts and a graduate of the Czechitas Digital Academy: Web. The skills she gained during the course, as well as her artistic sense, Zuzana now uses at Mild Blue, where she works as a junior frontend developer.",
```

Comment:
Jenom AJ grammar
The skills she gained during the course, as well as her artistic sense, Zuzana now uses at Mild Blue, where she works as a junior frontend developer.

->

She now uses the skills gained during the course as well as her artistic sense at Mild Blue, where she works as a junior frontend developer.